service/src/main/java/com/fecklessweasel/service/objectmodel/StoredFile.java

```diff
@@ -132,7 +136,7 @@ public static StoredFile create(Connection sql,
 
         // Write metadata to the database.
         Date creationDate = new Date();
-        int fid = addToDatabase(sql, user, course, title, description, creationDate);
+        int fid = addToDatabase(sql, user, course, title, description, creationDate, "notes");
```

Comment:
Is "notes" a hardcoded tag because we haven't wired this up to the front end yet?